Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -207,15 +207,15 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
         else { return }
 
         sections += [
-            (section: .name("Safe Name"), items: [Section.Name.name(safe.name ?? "Safe \(safe.addressValue.ellipsized())")]),
+            (section: .name("Safe Account Name"), items: [Section.Name.name(safe.name ?? "Safe \(safe.addressValue.ellipsized())")]),
 
             (section: .requiredConfirmations("Required confirmations"),
              items: [Section.RequiredConfirmations.confirmations("\(threshold) out of \(ownersInfo.count)")]),
 
             (section: .ownerAddresses("Owners"),
              items: ownersInfo.map { Section.OwnerAddresses.ownerInfo($0) }),
 
-            (section: .safeVersion("Safe version"),
+            (section: .safeVersion("Safe Account version"),
```

Comment:
On Android "Safe Account Base contract version" was used as a mastercopy version field name . Please. use the same names on both platforms